.rubocop.yml

```diff
@@ -50,6 +50,15 @@ Style/SymbolArray:
 Style/WordArray:
   EnforcedStyle: brackets
 
+# At the moment, the default confguration of the previous cops are pending
```

Comment:
Typo in conf**i**guration. Same in the commit description.

Reply:
Done